src/components/GameView/ScrapDialog.vue

```diff
@@ -24,7 +24,6 @@
       <v-card-actions class="d-flex justify-end my-2">
         <v-btn
           color="primary"
-          variant="outlined"
```

Comment:
This one can still be outlined. It's really the icon buttons we want to update here